src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_github.py

```diff
@@ -53,7 +53,7 @@ def create_webhook_when_required(scope, pipeline, map_params):
                 .get('properties', {})
                 .get('branch')
             )
-            or 'master'
```

Comment:
Does this need to be backwards compatible ```... or 'main' or 'master'```

Reply:
With v4.0.0 the default branch will change to 'main'.
I will add a CHANGELOG.md file describing these breaking changes shortly. So we can write-up the breaking changes ahead of the release.